addons/binding/org.openhab.binding.nikohomecontrol/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="nikohomecontrol" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd"><!--Niko Home Control Gateway Bridge Type -->
+
+	<bridge-type id="bridge">
+		<label>Niko Home Control Bridge</label>
+		<description>This bridge represents a Niko Home Control IP-interface</description>
+		<config-description>
+			<parameter name="ADDR" type="text" required="false">
+				<label>IP or Host Name</label>
+				<description>Address will be resolved on connection if not configured</description>
+				<advanced>true</advanced>
```

Comment:
@kaikreuzer I am afraid I will need more help here. I can move the broadcastAddr parameter to the binding. But I don't know how to access it from the bridge discovery service then. From what I have seen, I can easily access it from the handler factory by overriding the activate method and adding componentContext.getProperties(). But that does not seem to help me with the bridge discovery.
I agree I would be in trouble if there are 2 gateways on the local network. However, I think this is highly unlikely. I even suspect the whole Niko system would not work if that was the case. However, I did run into the issue I have 2 separate network interfaces on my openHAB system, therefore making it impossible reliably find the proper broadcast address automatically. I need to provide it. Once I have that, it is stable. The specific IP address is not stable though and my router does not allow me to fix it. So I cannot provide a fixed IP.
I am open to any guidance on how to do this better and indeed move the broadcast parameter to the binding level. Would that mean I have to register the discovery service from the handler factory, rather than through the declaration?